pom.xml

```diff
@@ -146,7 +146,7 @@
         <framework.features.tesb.version>4.2.7.tesb3</framework.features.tesb.version>
         <static.features.tesb.version>4.2.7.tesb1</static.features.tesb.version>
         <spring.features.tesb.version>4.2.7.tesb2</spring.features.tesb.version>
-        <spring-legacy.features.tesb.version>4.2.7.tesb2</spring-legacy.features.tesb.version>
+        <spring-legacy.features.tesb.version>4.2.7.tesb3</spring-legacy.features.tesb.version>
```

Comment:
Feature file is enterprise-legacy, not spring-legacy

Reply:
updated.